### PR TITLE
docs: More changes while translating (smoe:docs_man_adoc_20240918)

### PR DESCRIPTION
**Pull request by smoe**
A bit more while translating.

### CODE REVIEW

File: docs/src/gui/filter-programs.adoc [hunk 1]
```diff
@@ -66,8 +66,8 @@ This feature should be used by any filter that runs for a long time.
 
 Here is a very basic example of the filtering mechanics:
 When run through a Linucnc screen that offers program filtering,
-it will produce and write a line of G-code every 100th of a second to standard output.
-It also sends a progress message out to the Unix standard error stream.
+it will produce and write a line of G-code every 100^th^ of a second to standard output.
```
Comment (petterreinholdtsen) [inline docs/src/gui/filter-programs.adoc:69]: Why do you want to raise 'th' here?

**Comment by petterreinholdtsen** [comment]
A bit to long for me to review in one sitting, but I skimmed through it and found a few minor issues.  Besides this, it look good.

File: docs/src/gui/filter-programs.adoc (hunk 1 above)
Comment (smoe) [reply docs/src/gui/filter-programs.adoc:69]: That is where it should always be, like 1<sup>st</sup> or 2<sup>nd</sup>.

File: docs/src/gui/qtdragon.adoc [hunk 2]
```diff
@@ -946,7 +946,7 @@ This feature assumes the use of two probes in concert:
  . A tool switch sensor, fixed somewhere on the machine (sometimes called a tool-setter), and
  . A spindle probe that is installed temporarily at the beginning of the job (sometimes called an xyz probe or a Renishaw probe).
 
-These techniques are useful for machines that do not have repeatable tool holders and do not have automatic tool changing devices. (For machines with repeatable tool holders, see the section on <<sub:touch-plate,measuring tool length>>. For machines with automatic tool changing devices, consult work done under the linuxcnc repository at configs/sim/axis/remap/rack-toolchange)
+These techniques are useful for machines that do not have repeatable tool holders and do not have automatic tool changing devices. (For machines with repeatable tool holders, see the section on <<sub:touch-plate,measuring tool length>>. For machines with automatic tool changing devices, consult work done under the linuxcnc repository at configs/sim/axis/remap/rack-toolchange .)
```
[the remark this answers is not in the document]
Comment (smoe) [reply docs/src/gui/qtdragon.adoc:949]: Mistake.

**Comment by petterreinholdtsen** [comment]
[Steffen Möller]
> That is where it should always be, like 1<sup>st</sup> or
> 2<sup>nd</sup>.

Aha.  Did not know.  I guess it must be in the Chicago style, as it is
not in the Cambridge one, as far as I can tell.
<URL: https://dictionary.cambridge.org/grammar/british-grammar/dates >.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by smoe** [comment]
> [Steffen Möller]
> That is where it should always be, like 1<sup>st</sup> or 2<sup>nd</sup>.
> Aha. Did not know. I guess it must be in the Chicago style, as it is not in the Cambridge one, as far as I can tell. <URL: https://dictionary.cambridge.org/grammar/british-grammar/dates >.
> […](#)

It is indeed optional ... https://proofed.com/writing-tips/when-to-use-superscript-and-subscript-in-your-writing/ ... just some early brainwash.

File: docs/src/gui/qtdragon.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (smoe) [reply docs/src/gui/qtdragon.adoc]: Addressed in latest upload (also curious to learn if build is fine again).

**Comment by smoe** [comment]
This was a timeout error in the tests.
```
Running test: ./tests/single-step
*** ./tests/single-step: XFAIL: test run exited with 1
************** result:
        Machine configuration file is 'test.ini'
        Starting LinuxCNC...
        linuxcncsvr (216869) emcsvr: machine 'unknown'  version '1.1'
        linuxcnc TPMOD=tpmod HOMEMOD=homemod EMCMOT=motmod
        milltask (216881) task: machine 'unknown'  version '1.1'
        emc/task/taskclass.cc 187: can't load tool table.
        Found file(LIB): /usr/share/linuxcnc/hallib/core_sim.hal
        USRMOT: ERROR: command 30 timeout
        emcMotionInit: emcTrajInit failed
        task: main loop took 0.018273 seconds
        task: main loop took 0.018519 seconds
        Shutting down and cleaning up LinuxCNC...
        task: 0 cycles, min=0.000004, max=0.018519, avg=-1.000000, 2 latency excursions (> 10x expected cycle time of 0.001000s)
        LinuxCNC terminated with an error.  You can find more information in the log:
            /home/testrunner/linuxcnc_debug.txt
        and
            /home/testrunner/linuxcnc_print.txt
        as well as in the output of the shell command 'dmesg' and in the terminal
************** stderr:
        + linuxcnc -r test.ini
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        Note: Using POSIX non-realtime
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        emc/tooldata/tooldata_common.cc, 304: Failed to open tool table file 'simpockets.tbl': No such file or directory
        emc/tooldata/tooldata_common.cc, 304: Failed to open tool table file 'simpockets.tbl': No such file or directory
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        link (updating variable file): No such file or directory
        line=7; Xpos=5.00; Ypos=0.11; Zpos=0.00; counter=0
        line=7; Xpos=5.00; Ypos=0.94; Zpos=0.00; counter=0
        line=7; Xpos=5.00; Ypos=1.78; Zpos=0.00; counter=0
        line=7; Xpos=5.00; Ypos=2.61; Zpos=0.00; counter=0
        line=7; Xpos=5.00; Ypos=3.44; Zpos=0.00; counter=0
        line=7; Xpos=5.00; Ypos=4.28; Zpos=0.00; counter=0
        Taking step from X5.00 Y5.00
        line=9; Xpos=5.14; Ypos=5.00; Zpos=0.00; counter=0
        line=9; Xpos=5.98; Ypos=5.00; Zpos=0.00; counter=0
        line=9; Xpos=6.81; Ypos=5.00; Zpos=0.00; counter=0
        line=9; Xpos=7.64; Ypos=5.00; Zpos=0.00; counter=0
        line=9; Xpos=8.48; Ypos=5.00; Zpos=0.00; counter=0
        line=9; Xpos=9.31; Ypos=5.00; Zpos=0.00; counter=0
        Taking step from X10.00 Y5.00
        line=7; Xpos=10.00; Ypos=5.13; Zpos=0.00; counter=5
        line=7; Xpos=10.00; Ypos=5.97; Zpos=0.00; counter=5
        line=7; Xpos=10.00; Ypos=6.80; Zpos=0.00; counter=5
        line=7; Xpos=10.00; Ypos=7.63; Zpos=0.00; counter=5
        line=7; Xpos=10.00; Ypos=8.47; Zpos=0.00; counter=5
        line=7; Xpos=10.00; Ypos=9.30; Zpos=0.00; counter=5
        Taking step from X10.00 Y10.00
        line=9; Xpos=10.14; Ypos=10.00; Zpos=0.00; counter=5
        line=9; Xpos=10.98; Ypos=10.00; Zpos=0.00; counter=5
        line=9; Xpos=11.81; Ypos=10.00; Zpos=0.00; counter=5
        line=9; Xpos=12.64; Ypos=10.00; Zpos=0.00; counter=5
        line=9; Xpos=13.48; Ypos=10.00; Zpos=0.00; counter=5
        line=9; Xpos=14.31; Ypos=10.00; Zpos=0.00; counter=5
        Taking step from X15.00 Y10.00
        line=7; Xpos=15.00; Ypos=10.14; Zpos=0.00; counter=10
        line=7; Xpos=15.00; Ypos=10.98; Zpos=0.00; counter=10
        line=7; Xpos=15.00; Ypos=11.81; Zpos=0.00; counter=10
        line=7; Xpos=15.00; Ypos=12.64; Zpos=0.00; counter=10
        line=7; Xpos=15.00; Ypos=13.48; Zpos=0.00; counter=10
        line=7; Xpos=15.00; Ypos=14.31; Zpos=0.00; counter=10
        Taking step from X15.00 Y15.00
        line=9; Xpos=15.14; Ypos=15.00; Zpos=0.00; counter=10
        line=9; Xpos=15.98; Ypos=15.00; Zpos=0.00; counter=10
        line=9; Xpos=16.81; Ypos=15.00; Zpos=0.00; counter=10
        line=9; Xpos=17.64; Ypos=15.00; Zpos=0.00; counter=10
        line=9; Xpos=18.48; Ypos=15.00; Zpos=0.00; counter=10
        line=9; Xpos=19.31; Ypos=15.00; Zpos=0.00; counter=10
        Taking step from X20.00 Y15.00
        line=7; Xpos=20.00; Ypos=15.13; Zpos=0.00; counter=15
        line=7; Xpos=20.00; Ypos=15.97; Zpos=0.00; counter=15
        line=7; Xpos=20.00; Ypos=16.80; Zpos=0.00; counter=15
        line=7; Xpos=20.00; Ypos=17.63; Zpos=0.00; counter=15
        line=7; Xpos=20.00; Ypos=18.47; Zpos=0.00; counter=15
        line=7; Xpos=20.00; Ypos=19.30; Zpos=0.00; counter=15
        Taking step from X20.00 Y20.00
        line=9; Xpos=20.14; Ypos=20.00; Zpos=0.00; counter=15
        line=9; Xpos=20.98; Ypos=20.00; Zpos=0.00; counter=15
        line=9; Xpos=21.81; Ypos=20.00; Zpos=0.00; counter=15
        line=9; Xpos=22.64; Ypos=20.00; Zpos=0.00; counter=15
        line=9; Xpos=23.48; Ypos=20.00; Zpos=0.00; counter=15
        line=9; Xpos=24.31; Ypos=20.00; Zpos=0.00; counter=15
        Taking step from X25.00 Y20.00
        error: timeout in wait_complete_step(), 15.0 seconds were not enough
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        ERROR: ld.so: object 'libeatmydata.so' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
        Note: Using POSIX non-realtime
Running test: ./tests/source.0
**************
```

I suggest to accept the PR despite this unrelated build failure. Maybe the timeout can be extended?

**Comment by smoe** [comment]
Seems like our CI is a bit fuzzy. Worked now and - @petterreinholdtsen or @hansu, please give it a spin.

**Comment by petterreinholdtsen** [comment]
@smoe Please stop making these long patches with lots of unrelated changes lumped together.  They are a horrible task to review, and take so long I rarely are able to put aside the time required to do so, causing unreasonable delay for you waiting for someone to have time to look through it all.  Please split them into similar or related changes, making it easier to check and quicker to review.  It took me more than an hour to get through all these changes, to see if I found any errors, and I still have probably missed some mistakes.

In any case, I could not find any issues I believe need to be fixed now, the change look good, and I will squash and merge it.

**Pull request merged**